pkg/dns/vips/virtual_outbound_view.go

```diff
@@ -46,7 +46,11 @@ func (vo *VirtualOutboundMeshView) Add(entry HostnameEntry, outbound OutboundEnt
 			if existingOutbound.String() == outbound.String() {
 				return nil
 			}
-			return fmt.Errorf("can't add %s:%d from %s because it's already used by entity defined in:'%s'", entry.Name, outbound.Port, outbound.Origin, existingOutbound.Origin)
+			if entry.Type == Host {
+				return fmt.Errorf("autogenerated DNS entry %s:%d conflicts with existing entry, this happens when there are two external services with the same 'networking.address', to disable automatic DNS generation in external services set 'skipVIPGeneration' to 'true' under 'networking'", entry.Name, outbound.Port)
```

Comment:
```suggestion
				return fmt.Errorf("autogenerated DNS entry %s:%d conflicts with existing entry, this happens when there are two external services with the same 'networking.address', to disable automatic DNS generation in external services set 'networking.skipVIPGeneration=true' in at least one of these externalService", entry.Name, outbound.Port)
```

Reply:
Should you add `outbound.Origin` to get the externalService name?

Reply:
If you do this you should fix:

https://github.com/kumahq/kuma/blob/826cd4c68f10e2a2b812a0e8c351d3606f3cad75/pkg/dns/vips_allocator.go#L233-L237

To add the external service in the origin just like we do with virtualoutbound:

https://github.com/kumahq/kuma/blob/826cd4c68f10e2a2b812a0e8c351d3606f3cad75/pkg/dns/vips_allocator.go#L296-L300

https://github.com/kumahq/kuma/blob/826cd4c68f10e2a2b812a0e8c351d3606f3cad75/pkg/dns/vips/virtual_outbound.go#L34


It will ease up user's figuring out which ES is causing the problem

Reply:
I'm adding the externalService name here - https://github.com/kumahq/kuma/pull/5185/files#diff-ad441c05e85b5598449acabae1ac6a08e6dcd3017ec295b06bfa87f4633c05edR241 one level up, do you think it's better placed here?

Reply:
I think in the origin it's better because it's persisted in many different places (.e.g there's a configMap where you can see this origin).